tests/functional/WebDriverWindowTest.php

```diff
@@ -48,6 +50,34 @@ public function testShouldMaximizeWindow()
         $this->assertGreaterThanOrEqual($sizeBefore->getHeight(), $sizeAfter->getHeight());
     }
 
+    /**
+     * @group exclude-chrome
+     */
+    public function testShouldMinimizeWindow()
+    {
+        self::skipForJsonWireProtocol('"minimize" is not supported in JsonWire Protocol');
+
+        $this->assertSame('visible', $this->driver->executeScript('return document.visibilityState;'));
+
+        $this->driver->manage()
+            ->window()
+            ->minimize();
+
+        $this->assertSame('hidden', $this->driver->executeScript('return document.visibilityState;'));
+    }
+
+    public function testShouldMinimizeWindowJsonWire()
```

Comment:
I don't think we need this kind of tests... They IMHO don't add a real value, only slow down the build - there is no interaction with the browser, dispite this the browser is started which takes eg. on SauceLabs 10+ seconds. 

So this tests which only tests internal logic could be unit tests, however, I think it is unnecessary altogether (considering we will ditch JsonWire support later).